packages/lib/modules/pool/pool.helpers.ts

```diff
@@ -420,33 +421,43 @@ export function getWarnings(warnings: string[]) {
   return warnings.filter(warning => !isEmpty(warning))
 }
 
+export function getActionableTokenSymbol(tokenAddress: Address, pool: Pool): string {
+  const token = getPoolActionableTokens(pool).find(token =>
+    isSameAddress(token.address, tokenAddress)
+  )
+  if (!token) {
+    console.log('Token symbol not found for address ', tokenAddress)
+    return ''
+  }
+
+  return token.symbol
+}
```

Comment:
Same comment, make it more generic, doesn't need to be limited to the action tokens.